paket.dependencies

```diff
@@ -20,14 +20,16 @@ group Scraper
 group Client
     source https://api.nuget.org/v3/index.json
 
-	nuget Fable.Core
+	nuget Fable.Core ~> 2.0.2
 	nuget Fable.React
-	nuget Fable.Elmish.React
-	nuget Fable.Elmish.Browser
+	nuget Fable.Elmish.React ~> 2.1
+	nuget Fable.Elmish.Browser ~> 2.1
 	nuget Fable.Elmish.Debugger
-	nuget Fable.Elmish.HMR
+	nuget Fable.Elmish.HMR ~> 2.0
```

Comment:
Why do we have to set the specific versions of dependencies?

Reply:
As far as I remember it fails to compile if we use the latest version instead. Fulma library has a dependency on that one, but doesn't have a requirement for specific version. Nevertheless, it fails to work with the newest version.

Reply:
Alright, let it be